src/maintainer/pinning_deps.rst

```diff
@@ -94,26 +99,9 @@ Updating package pins
 
 Updating a pin requires following steps:
 
- - edit `conda_build_config.yaml <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`__ in the ``conda-forge/conda-forge-pinning`` repository.
- - bump the version of the conda-forge-pinning `recipe <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/meta.yaml>`__ by setting the version to the current date.
- - rerender the feedstock.
+ - create a new migration yaml by copying `example.exyaml <https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/example.exyaml>`__ in the ``conda-forge/conda-forge-pinning`` repository.
+ - change the migration yaml to reflect the package and version to be migrated
  - propose the changes as a :term:`PR` to ``conda-forge/conda-forge-pinning-feedstock``.
- - write a :ref:`migrator <pin_migrator>` for propagating the pin changes.
-
-.. _pin_migrator:
-
-Propagate pin changes with a migrator
--------------------------------------
-
-Changing global pins requires rerendering all packages that depend on the package with the changed pin. Doing this manually can be tedious, especially when many packages are involved.
-Migrators are used to automatically generate pull requests for the affected packages in conda-forge.
-
-Migrators are added to the `migrations folder in conda-forge-pinning-feedstock <https://github.com/conda-forge/conda-forge-pinning-feedstock/tree/master/recipe/migrations>`__.
-
-
-You can do this by forking `conda-forge/conda-forge-pinning-feedstock <https://github.com/conda-forge/conda-forge-pinning-feedstock>`__ and submitting a pull request.
-
-Details of how the migration yaml is setup are provided in an `example <https://github.com/conda-forge/conda-forge-pinning-feedstock/tree/master/recipe/migrations/example.exyaml>`__
-and documentation `here <https://regro.github.io/cf-scripts/migrators.html#building-a-migration-yaml>`_.
```

Comment:
Did we mean to drop this content? Is this somewhere else now?

Reply:
Hmm, sorry. I deleted the wrong thing. I'll restore this

Reply:
No worries. Thanks for the update 🙂